apps/web/src/state/farmsV4/state/accountPositions/fetcher.ts

```diff
@@ -153,11 +116,15 @@ const V2_UNIVERSAL_FARMS = UNIVERSAL_FARMS.filter((farm) => farm.protocol === Pr
 const STABLE_UNIVERSAL_FARMS = UNIVERSAL_FARMS.filter((farm) => farm.protocol === Protocol.STABLE)
 
 export const getBCakeWrapperAddress = (lpAddress: Address, chainId: number) => {
-  const f = UNIVERSAL_BCAKEWRAPPER_FARMS.find((farm) => {
+  const f = UNIVERSAL_FARMS.find((farm) => {
```

Comment:
There are very few farms with bCake, so iterating through them each time would lead to some inefficiency. How about caching a `farmsWithBCake` array filtered from `UNIVERSAL_FARMS` and can be reused here

Reply:
use map instead